src/app/ui/components/spinner.tsx

```diff
@@ -9,7 +9,7 @@ interface SpinnerProps extends SquareProps {
 
 export function Spinner({
   emptyColor = 'transparent',
-  size = '1.5rem',
+  size = '16px',
```

Comment:
Maybe we can have a token for this?

Reply:
Yeah, I should change this to `sm`, but getting a little annoyed with the `SizeToken` not working. Haha.